src/plugins/expression_shape/common/types/index.ts

```diff
@@ -5,5 +5,21 @@
  * in compliance with, at your election, the Elastic License 2.0 or the Server
  * Side Public License, v 1.
  */
-export * from './expression_functions';
-export * from './expression_renderers';
+
+export type {
```

Comment:
I think it's fine to do `export *` in places as long as it's not a root index file.  No need to change anything, just a heads up